components/usage/pkg/db/workspace_instance.go

```diff
@@ -59,8 +59,9 @@ func ListWorkspaceInstancesInRange(ctx context.Context, conn *gorm.DB, from, to
 		Where(
 			conn.Where("stoppedTime >= ?", from).Or("stoppedTime = ?", ""),
 		).
-		Where("creationTime < ?", to).
-		Where("creationTime != ?", "").
```

Comment:
This query would not match records without a creation time as `creationTime` cannot be both < `to` and `empty`